sympy/functions/elementary/integers.py

```diff
@@ -268,11 +274,17 @@ def _eval_number(cls, arg):
             return arg.approximation_interval(Integer)[1]
 
     def _eval_nseries(self, x, n, logx, cdir=0):
+        arg = self.args[0]
+        arg0 = arg.subs(x, 0)
+        if arg0.is_infinite:
+            from sympy.calculus.util import AccumBounds
+            from sympy.series.order import Order
+            s = arg._eval_nseries(x, n, logx, cdir)
+            o = Order(1, (x, 0)) if n <= 1 else AccumBounds(-1, 0)
```

Comment:
The value of `ceiling(x) - x` lies between 0 and 1..

Reply:
That was silly, didn't pay attention